controller/predicates/is_pip_service_enabled.js

```diff
@@ -0,0 +1,9 @@
+const _ = require('lodash');
+
+module.exports = (uri) => {
+  // this predicate relies upon the fact that the schema has already validated
+  //  that api.pipService is a URI-formatted string
+  return (request, response) => {
+    return uri !== undefined;
+  };
```

Comment:
isSet is the wrong function, sorry. It checks that the value is of type Set, which I always forget! but there is _.isUndefined()